tests/tasks/check_header.yml

```diff
@@ -9,8 +9,8 @@
 - name: Check for presence of ansible managed header, fingerprint
   assert:
     that:
-      - ansible_managed in content
+      - __ansible_managed in content
       - __fingerprint in content
   vars:
-    content: "{{ (__file_content | d(__content)).content | b64decode }}"
-    ansible_managed: "{{ lookup('template', 'get_ansible_managed.j2') }}"
+    content: "{{ __content.content | b64decode }}"
```

Comment:
yeah - I copied this implementation from another role where I already have the file content, so I could skip the `slurp` - but here it isn't used